bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/internal/DPIUtil.java

```diff
@@ -455,26 +455,26 @@ public static Point autoScaleUp(Point point, int zoom) {
  * Returns a new scaled up Point if enabled for Drawable class.
  */
 public static Point autoScaleUp(Drawable drawable, Point point) {
-	return autoScaleUp (drawable, point, deviceZoom);
+	return scaleUp (drawable, point, deviceZoom);
 }
 
-public static Point autoScaleUp(Drawable drawable, Point point, int zoom) {
+public static Point scaleUp(Drawable drawable, Point point, int zoom) {
 	if (drawable != null && !drawable.isAutoScalable()) return point;
-	return autoScaleUp (point, zoom);
+	return scaleUp (point, zoom);
 }
 
 /**
  * Returns a new scaled up Rectangle.
  */
 public static Rectangle autoScaleUp(Rectangle rect) {
-	return autoScaleUp(rect, deviceZoom);
+	return scaleUp(rect, deviceZoom);
 }
 
-public static Rectangle autoScaleUp(Rectangle rect, int zoom) {
+public static Rectangle scaleUp(Rectangle rect, int zoom) {
 	if (zoom == 100 || rect == null) return rect;
 	Rectangle scaledRect = new Rectangle(0,0,0,0);
-	Point scaledTopLeft = DPIUtil.autoScaleUp (new Point(rect.x, rect.y), zoom);
-	Point scaledBottomRight = DPIUtil.autoScaleUp (new Point(rect.x + rect.width, rect.y + rect.height), zoom);
+	Point scaledTopLeft = DPIUtil.scaleUp (new Point(rect.x, rect.y), zoom);
+	Point scaledBottomRight = DPIUtil.scaleUp (new Point(rect.x + rect.width, rect.y + rect.height), zoom);
```

Comment:
nitpicky:
```suggestion
	Point scaledTopLeft = scaleUp (new Point(rect.x, rect.y), zoom);
	Point scaledBottomRight = scaleUp (new Point(rect.x + rect.width, rect.y + rect.height), zoom);
```

Reply:
Done, also refactored the similar thing in a few more places.